### PR TITLE
Changes to properties and configuration to get conductor working with…

### PR DESCRIPTION
**Pull request by abhilasha1307**
… postgres persistence

Pull Request type
----
- [x ] Bugfix

Changes in this PR
----
Changes to properties and configuration to get conductor working with postgres persistence

Issue #

image
image (2)

image (3)
image (4)

### CODE REVIEW

**Comment by abhilasha1307** [comment]
Link to Issue : https://github.com/Netflix/conductor/issues/3591

File: docker/server/config/config-postgres.properties [hunk 1]
```diff
@@ -9,8 +9,8 @@ spring.datasource.username=conductor
 spring.datasource.password=conductor
 
 # Hikari pool sizes are -1 by default and prevent startup
-spring.datasource.hikari.maximum-pool-size=10
-spring.datasource.hikari.minimum-idle=2
+#spring.datasource.hikari.maximum-pool-size=10
```
Comment (manan164) [inline docker/server/config/config-postgres.properties:12]: Hi @abhilasha1307 , Please revert this change.

File: docker/server/config/config-postgres.properties [hunk 2]
```diff
@@ -22,4 +22,9 @@ conductor.elasticsearch.url=http://es:9200
 conductor.elasticsearch.indexName=conductor
 
 # Load sample kitchen sink workflow
-loadSample=true
+loadSample=false
```
Comment (manan164) [inline docker/server/config/config-postgres.properties:25]: Hi @abhilasha1307 , Please revert this change.

File: server/build.gradle [hunk 3]
```diff
@@ -44,6 +44,7 @@ dependencies {
     implementation "org.springdoc:springdoc-openapi-ui:${revOpenapi}"
 
     runtimeOnly "org.glassfish.jaxb:jaxb-runtime:${revJAXB}"
+    runtimeOnly 'com.netflix.conductor:conductor-postgres-persistence:3.13.5'
```
Comment (manan164) [inline server/build.gradle:47]: This will always include postgres as a dependency. We should only include when db.type is Postgres.

Comment (v1r3n) [reply server/build.gradle:47]: +1 to what @manan164 said.  This introduces the postgres dependency with a fixed version.
